### PR TITLE
Add digit captcha to the Call for paper form #57

### PR DESCRIPTION
**Pull request by Biletskiy**

### CODE REVIEW

File: src/Event/EventBundle/Form/Type/CallForPaperType.php [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 use Symfony\Component\Form\AbstractType;
 use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\FormEvent;
+use Symfony\Component\Form\FormEvents;
```
Comment (pilot) [inline src/Event/EventBundle/Form/Type/CallForPaperType.php]: what for added both use?

Comment (Biletskiy) [inline src/Event/EventBundle/Form/Type/CallForPaperType.php]: Plugin added

File: src/Event/EventBundle/Controller/EventController.php [hunk 2]
```diff
@@ -122,7 +123,9 @@ public function callForPaperAction(Request $request)
         }
 
         return new Response($this->renderView('EventEventBundle:Event:_form.html.twig', [
-            'form' => $form->createView()
+            'event' => $event,
```
Comment (pilot) [inline src/Event/EventBundle/Controller/EventController.php]: remove as not used

File: src/Event/EventBundle/Controller/EventController.php [hunk 3]
```diff
@@ -119,11 +119,24 @@ public function callForPaperAction(Request $request)
...
-            'event' => $event,
+            'form' => $form->createView(),
+            'captcha' => $this->getCaptcha()
+        ]));
+    }
+
+    public function callForPaperViewAction()
+    {
+        $entity = new CallForPaper();
+        $event = $this->getEvent();
```
Comment (pilot) [inline src/Event/EventBundle/Controller/EventController.php]: remove extra middle var

File: src/Event/EventBundle/Controller/EventController.php [hunk 4]
```diff
@@ -119,11 +119,24 @@ public function callForPaperAction(Request $request)
...
+        $entity->setEvent($event);
```
Comment (pilot) [inline src/Event/EventBundle/Controller/EventController.php]: just `$entity->setEvent($this->getEvent())`;

File: src/Event/EventBundle/Controller/EventController.php [hunk 5]
```diff
@@ -118,11 +119,25 @@ public function callForPaperAction(Request $request)
                 $form->get('email')->getData()
             );
 
-            return new Response('Success');
+                return new Response('Success');
```
Comment (pilot) [inline src/Event/EventBundle/Controller/EventController.php]: the wrong indentation put it back

**Pull request merged**